MediaBrowser.Api/Playback/BaseStreamingService.cs

```diff
@@ -289,16 +289,20 @@ private async Task AcquireResources(StreamState state, CancellationTokenSource c
                 throw;
             }
 
+            Logger.LogDebug("Launched ffmpeg process");
             state.TranscodingJob = transcodingJob;
 
             // Important - don't await the log task or we won't be able to kill ffmpeg when the user stops playback
             _ = new JobLogger(Logger).StartStreamingLog(state, process.StandardError.BaseStream, logStream);
 
             // Wait for the file to exist before proceeeding
-            while (!File.Exists(state.WaitForPath ?? outputPath) && !transcodingJob.HasExited)
+            var waitFor = state.WaitForPath ?? outputPath;
```

Comment:
Not a very descriptive variable name. What is it a path to?

Reply:
renamed to `ffmpegTargetFile`